packages/wallet/src/components/transfer/Badge.tsx

```diff
@@ -0,0 +1,13 @@
+type BadgeProps = {
+  text: string
+}
+
+export const Badge = ({ text }: BadgeProps) => {
+  return (
+    <>
+      <span className="relative top-3 inline-flex w-10 justify-center rounded-md bg-gradient-to-r from-[#92DBCA] to-[#56B1AF] text-sm text-white">
+        {text}
+      </span>
+    </>
```

Comment:
```suggestion
      <span className="inline-flex w-10 justify-center rounded-md bg-gradient-to-r from-[#92DBCA] to-[#56B1AF] text-sm text-white">
        {text}
      </span>
```

Also, I think we can move this in the `ui` folder.

Reply:
I added back the relative and top-3 to bring it closer to the Input, as it is in design, othervise it will be far away from Input